Makefile

```diff
@@ -1,8 +1,21 @@
-.PHONY: test run local
-
-test:
-	pytest
-run:
-	docker run --rm -it -v $(PWD)/notebooks:/notebooks -w /notebooks -p 8888:8888 openmined/pysyft jupyter notebook --ip=0.0.0.0 --allow-root
-custom:
-	docker run --rm -it -v $(PWD)/notebooks:/notebooks -w /notebooks -p 8888:8888 $(docker) jupyter notebook --ip=0.0.0.0 --allow-root
\ No newline at end of file
+.PHONY: install test notebook docker-build docker-run
+
+install:
+	python3 setup.py install
+
+test: install
+	pytest && pytest --flake8
+
+notebook:
+	jupyter notebook --allow-root --ip=0.0.0.0
+
+dockerfile = Dockerfile
```

Comment:
does this get overridden by make command parameters?

Reply:
Yes, variables in the makefile can be overridden, let me google [this](https://www.gnu.org/software/make/manual/html_node/Overriding.html) for you